openeo-geotrellis/src/main/scala/org/openeo/geotrellis/package.scala

```diff
@@ -89,4 +90,32 @@ package object geotrellis {
 
     Region.of(regionName)
   }
+
+  def toSigned(cellType: CellType): CellType = {
+    cellType match {
+      case UByteCellType => ByteCellType
+      case UByteConstantNoDataCellType => ByteConstantNoDataCellType
+      case c: UByteUserDefinedNoDataCellType => ByteUserDefinedNoDataCellType(c.noDataValue)
```

Comment:
No need to introduce a name here, just leverage pattern matching:

```scala
case UByteUserDefinedNoDataCellType(noDataValue) => ByteUserDefinedNoDataCellType(noDataValue)
```

Reply:
Nice syntax. Changed